nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/recompute/RecomputeViewsActionValidation.java

```diff
@@ -0,0 +1,24 @@
+package org.nuxeo.ecm.platform.picture.recompute;
+
+import static org.nuxeo.ecm.platform.picture.recompute.RecomputeViewsAction.PARAM_XPATH;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.nuxeo.ecm.core.bulk.AbstractBulkActionValidation;
+import org.nuxeo.ecm.core.bulk.message.BulkCommand;
+
+public class RecomputeViewsActionValidation extends AbstractBulkActionValidation {
```

Comment:
` @since 11.1`

Reply:
Could we have a test class for the validation ( like `TestSetPropertiesValidation` for example) ?